src/test/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReaderTest.java

```diff
@@ -124,4 +129,40 @@ private void assertDoesNotHaveVariantAnnotation(VariantWrapper variantWrapper)
         VariantAnnotation annotation = ((Variant) privateVariantField.get(variantWrapper)).getAnnotation();
         assertNull(annotation.getConsequenceTypes());
     }
+
+    @Test
+    public void shouldReadVariantsInAllStudies() throws Exception {
```

Comment:
This method tests against a single study, doesn't it? The name sounds a bit weird when applied to a singular count.